pgx/src/datum/into.rs

```diff
@@ -45,6 +45,23 @@ where
     }
 }
 
+impl<T, E> IntoDatum for Result<T, E>
```

Comment:
This is clearly an accidental addition.

Reply:
Can you tell what I was thinking about last night? Ha.